oak_functions/loader/src/metrics.rs

```diff
@@ -21,11 +21,12 @@ use crate::{
         OakApiNativeExtension, WasmState, ABI_USIZE,
     },
 };
+use alloc::sync::Arc;
 use oak_functions_abi::proto::OakStatus;
 use oak_functions_metrics::{
     PrivateMetricsAggregator, PrivateMetricsConfig, PrivateMetricsExtension, PrivateMetricsProxy,
 };
-use std::sync::{Arc, Mutex};
+use oak_functions_util::sync::Mutex;
```

Comment:
As mentioned in the email thread, I think it might be useful in future to feature-gate this for `no_std` and `parking_lot` in environments where `std` is available. Perhaps create an issue for this and add a TODO here?

Reply:
Created a new issue and added the comment.